dynamiq/connections/connections.py

```diff
@@ -1030,3 +1030,19 @@ def conn_params(self) -> dict:
         return {
             "api_base": self.url,
         }
+
+
+class Jina(Http):
+    """
+    Connection class for Jina Scrape API.
+    """
+
+    api_key: str = Field(default_factory=partial(get_env_var, "JINA_API_KEY"))
+    method: str = HTTPMethod.GET
```

Comment:
it seems here method shouldn't be changed
```suggestion
    method: Literal[HTTPMethod.GET] = HTTPMethod.GET
```